src/main/webapp/usermap.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link rel="stylesheet" href="stylesheets/teammates.css" type="text/css">
+    <meta name="application-name" content="TEAMMATES - Online Peer Feedback/Evaluation System for Student Team Projects">
+    <meta name="keywords" content="Teammates, peer feedback, peer evaluation, student, team, project, free, user, map">
+    <meta name="description" content="TEAMMATES is an Online Peer Feedback System for student team projects. It is completely free to use.
+    Get your students to evaluate their performance in team projects, and view reports and summaries of their feedback and evaluations.">
+    <title>TEAMMATES[User Map] - Online Peer Feedback/Evaluation System for Student Team Projects<</title>
+    <link rel="apple-touch-icon" href="apple-touch-icon.png" />
+    <script type="text/javascript" src="/js/googleAnalytics.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/d3/3.5.3/d3.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/topojson/1.6.9/topojson.min.js"></script>
+    <script src="/js/lib/datamaps.world.min.js"></script>
+    <script src="/js/countryCodes.js"></script>
+    <script src="/js/userMapData.js"></script>
+    <script src="/js/userMap.js"></script>
+</head>
+
+<body>
+    <div id="mainContainer">
+        <div id="header">
+            <div id="headerWrapper">
+                <div id="imageHolder">
+                    <a href="/index.html">
+                    <img alt="TEAMMATES[Logo] - Online Peer Feedback/Evaluation System for Student Team Projects" src="images/teammateslogo.jpg" width="150px" height="47px">
+                    </a>
+                </div>
+                <div id="menuHolder">
+                    <div id="textHolder">
+                        <ul id="navbar">
+                            <li><a href="index.html">Home</a></li>
+                            <li><a href="features.html">Features</a></li>
+                            <li><a href="about.html">About Us</a></li>
+                            <li><a href="contact.html">Contact</a></li>
+                            <li><a href="terms.html">Terms of Use</a></li>
+                        </ul>
+                    </div>
+                    <div id="loginHolder">
+                            <form action="/login" style="float:left" name="studentLogin">
+                             <input type="submit" name="student" class="button" id="btnStudentLogin" value="Student Login">
+                         </form>
+                         <form action="/login" name="instructorLogin" style="float:left">
+                             <input type="submit" name="instructor" class="button" id="btnInstructorLogin" value=" Instructor Login">
+                         </form>
+                    </div>
+                    <div style="clear: both;"></div>
+                </div>
+                <div style="clear: both;"></div>
```

Comment:
It was present in all the other html files so I thought it might be there for a reason.